infrastructure/docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+services:
+  buns_service:
+    container_name: buns_store
+    build:
+      context: ../
+      dockerfile: infrastructure/buns_store/service/Dockerfile
+    ports:
+      - 8020:8020
+
+  bakery_db:
+    container_name: bakery_db
+    build:
+      context: ./bakery_service/database
+      dockerfile: Dockerfile
+    ports:
+      - 5555:5432
+    environment:
+      - POSTGRES_PASSWORD=foo
+      - POSTGRES_USER=bar
+
+  bakery_service:
+    container_name: bakery_service
```

Comment:
Does this work with name consisting of underscore `_`? :)  Please see my last email and let me know if the tips included there worked:)